woocommerce/woocommerce-android
# Refund by items: Full refund flow - Step 6

This PR brings the complete refund-by-items flow. 

To test:

1. Create an order with multiple items
2. Go to order detail and issue a refund
3. Select a subset of items for refund 
4. **Verify the subtotal, total and tax calculations are correct**
5. Finish the partial refund and notice a new refund item appears in the order detail
6. **On the web, verify that the number of refunded items in order detail matches the recent refund quantities**
7. Go back to the app and create a full refund by selecting all items
8. Complete the refund and notice a new refund item appears in the order detail
9. **Verify the Issue refund button is not visible anymore and the order status is now Refunded**
10. **Verify that the refund tab changes, number picker, select all button taps are all tracked**
